draft-ietf-tls-esni.md

```diff
@@ -930,6 +959,28 @@ in-band when possible, such as through the use of OCSP stapling, and clients
 SHOULD take steps to minimize or protect such requests during certificate
 validation.
 
+## Abuse of ECH Acceptance Signal
+
+To signal acceptance, the backend server overwrites 8 bytes of its
+ServerHello.random with a value derived from the ClientHelloInner.random. (See
+{{backend-server-behavior}} for details.) In the presence of an active
+adversary, this behavior increases the likelihood of the ServerHello.random
+colliding with the ServerHello.random of a previous session, potentially
+reducing the overall security of the protocol. However, the remaining 24 bytes
+provide enough entropy to ensure this is not a practical avenue of attack.
+
+On the other hand, the probability that two 8-byte strings are the same is
+non-negligible. This poses a modest operational risk. Suppose the client-facing
+server terminates the connection (i.e., ECH is rejected or bypassed): if the
+last 8 bytes of its ServerHello.random coincide with the confirmation signal,
+then the client will incorrectly presume acceptance and proceed as if the
+backend server terminated the connection.
+
+Note that the same bytes of the ServerHello.random are used to signal to the
+client that an earlier version of TLS was negotiated (see {{RFC8446}}, Section
+4.1.3). The backend server's signal of acceptance does not interfere with this
+mechanism because ECH is only supported in TLS 1.3 or higher.
```

Comment:
```suggestion
Note that the same bytes of the ServerHello.random are used to implement
downgrade protection for TLS 1.3 (see {{RFC8446}}, Section
4.1.3). The backend server's signal of acceptance does not interfere with this
mechanism because ECH is only supported in TLS 1.3 or higher.
```

Reply:
Done.